test/e2e/user_workloads_test.go

```diff
@@ -34,7 +34,7 @@ func (s *userWorkloadsTestSuite) SetupSuite() {
 }
 
 func (s *userWorkloadsTestSuite) TearDownTest() {
-	s.ctx.Cleanup()
+	// s.ctx.Cleanup()
```

Comment:
Forgot to uncomment?

Reply:
ah right! 🤦‍♂️

Reply:
one day I will code a linter to check that `ctx.Cleanup` is not commented out in my PRs! 😂